index.html

```diff
@@ -120,11 +118,10 @@ <h4>"Public Domanin Audio Book"</h4>
           <source
             type="image/svg+xml"
             srcset="Images/sc-logo.svg"
-            media="(max width:70px)"
           />
-          <img
+          <img width="70" alt="Sault College logo"
```

Comment:
Just use the width attribute.  (max-width:70px) is a media condition that checks if the browser is under 70px wide